presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -517,17 +530,33 @@ std::shared_ptr<protocol::CallExpression> isNot(
   return isFunctionCall(expression, kNot);
 }
 
-/// Check if input RowExpression is an 'a > b' expression and returns it as
-/// CallExpression. Returns nullptr if input expression is something else.
+// Check if input RowExpression is an 'a > b' expression and returns it as
+// CallExpression. Returns nullptr if input expression is something else.
 std::shared_ptr<protocol::CallExpression> isGreaterThan(
     const std::shared_ptr<protocol::RowExpression>& expression) {
-  static const std::string_view kGreaterThan =
+  static constexpr std::string_view kGreaterThan =
       "presto.default.$operator$greater_than";
   return isFunctionCall(expression, kGreaterThan);
 }
 
-/// Checks if input PlanNode represents a local exchange with single source and
-/// returns it as ExchangeNode. Returns nullptr if input node is something else.
+// Check if input RowExpression is a 'between' expression and returns it as
+// CallExpression. Returns nullptr if input expression is something else.
+std::shared_ptr<protocol::CallExpression> isBetween(
```

Comment:
```
s/isBetween/isBetweenFunctionCall/
```

Reply:
Let's keep the naming consistent with the existing ones? like `isNot`, `isGreaterThan`.